ssn/integrated/sosa.ttl

```diff
@@ -34,19 +34,45 @@ cc:licence a owl:AnnotationProperty .
 vann:preferredNamespacePrefix a owl:AnnotationProperty .
 vann:preferredNamespaceUri a owl:AnnotationProperty .
 vs:term_status a owl:AnnotationProperty .
-<http://meta.schema.org/domainIncludes> a owl:AnnotationProperty .
-<http://meta.schema.org/rangeIncludes> a owl:AnnotationProperty .
+schema:domainIncludes a owl:AnnotationProperty .
+schema:rangeIncludes a owl:AnnotationProperty .
```

Comment:
use schema:domainIncludes instead of <http://meta.schema.org/domainIncludes>, this is already corrected in sosa.ttl
use schema:rangeIncludes instead of <http://meta.schema.org/rangeIncludes>, this is already corrected in sosa.ttl